inksdk/inksample/build.gradle

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License.
+ */
+
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+}
+
+android {
```

Comment:
same here than in the `build.gradle` library file, please, avoid using hardcoded values and instead reference the ones located in the `dependencies.gradle` file. Example here: https://github.com/microsoft/surface-duo-sdk/blob/main/bottomnavigation/sample/build.gradle